allennlp/nn/util.py

```diff
@@ -2020,7 +2021,7 @@ def tiny_value_of_dtype(dtype: torch.dtype):
 _V = TypeVar("_V", int, float)
 
 
-def dist_reduce(value: _V, reduce_op: ReduceOp, **kwargs) -> _V:
+def dist_reduce(value: _V, reduce_op: dist.ReduceOp = dist.ReduceOp.SUM, **kwargs) -> _V:
```

Comment:
Deferred type annotations also works on default arguments?

Reply:
Oops, I forgot to change this back

Reply:
Fixed.